index.html

```diff
@@ -9,59 +9,65 @@
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="viewport" content="width=device-width" />
 
-<link rel="icon" href="https://delta.chat/assets/logos/delta-chat.svg" sizes="any" type="image/svg+xml">
+<link rel="icon" href="logo.svg" sizes="any" type="image/svg+xml">
 
 <base target="_blank">
 
 <style>
 body {
-  font-family: 'Lucida Grande', sans-serif;
-  font-size: 10pt;
-
-  max-width: 420px;
-
-  margin: auto;
-  margin-top: 0;
-
-  background: linear-gradient(120deg, rgb(119, 136, 143), rgb(54, 78, 89));
+  margin: 0px;
+  padding: 0px;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  background-color: #000;
+  color: #bbb;
+  font-size: 14px;
+  font-weight: light;
```

Comment:
```suggestion
  font-weight: lighter;
```

I think this is a typo

Reply:
i thought `light` exists as well, but you're right, [it is just `lighter`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) :)